tests/failpoints/cases/test_pd_client_legacy.rs

```diff
@@ -245,8 +246,8 @@ fn test_reconnect_limit() {
     let pd_client_reconnect_fp = "pd_client_reconnect";
     let (_server, client) = new_test_server_and_client(ReadableDuration::secs(100));
 
-    // The GLOBAL_RECONNECT_INTERVAL is 0.1s so sleeps 0.2s here.
-    thread::sleep(Duration::from_millis(200));
+    // The default retry interval is 1s so sleeps 2s here.
```

Comment:
Here I don't change it since we need to rewrite `new_test_server_and_client`, I think it's ok to keep it.